basicsr/data/degradations.py

```diff
@@ -5,7 +5,7 @@
 import torch
 from scipy import special
 from scipy.stats import multivariate_normal
-from torchvision.transforms.functional_tensor import rgb_to_grayscale
+from torchvision.transforms.functional import rgb_to_grayscale
```

Comment:
this will work--
try:
    from torchvision.transforms.functional import rgb_to_grayscale
except ImportError:
    from torchvision.transforms.functional_tensor import rgb_to_grayscale